pkg/backup/backup/backup_manager.go

```diff
@@ -244,7 +244,7 @@ func (bm *backupManager) checkVolumeBackupInitializeJobRunning(backup *v1alpha1.
 		// all the volume snapshots has created
 		return nil
 	}
-	if !v1alpha1.IsVolumeBackupInitialized(backup) || v1alpha1.IsVolumeBackupInitializeFailed(backup) {
+	if v1alpha1.IsVolumeBackupInitializeFailed(backup) {
```

Comment:
Let me understand the removal of IsVolumeBackupInitialized. Now if the init job is stuck and eventually ttled, then  this method won't find the job, and return "init job deleted before ..." error here https://github.com/pingcap/tidb-operator/blob/6ec83b038432fd5c64458e883d7c12d02e83b5d3/pkg/backup/backup/backup_manager.go#L268. Right?

Reply:
No, the job still exists but it has failed status due to ttl. So we will find job failed and set backup failed here https://github.com/pingcap/tidb-operator/blob/6ec83b038432fd5c64458e883d7c12d02e83b5d3/pkg/backup/backup/backup_manager.go#L292

In addition, I verified the change by unit test. We can't remove the condition directly because in the first reconcile, the backup job is not created and we will get a 'not found' error. So I modified the condition.